### PR TITLE
[DOCS] Update transform and anomaly detection rule creation steps

### PR DESCRIPTION
**Pull request by lcawl**
Relates to https://github.com/elastic/kibana/issues/175634

This PR updates https://www.elastic.co/guide/en/elasticsearch/reference/current/transform-alerts.html and https://www.elastic.co/guide/en/machine-learning/current/ml-configuring-alerts.html
In particular, it removes screenshots that are no longer accurate or necessary.

### CODE REVIEW

File: docs/reference/transform/transform-alerts.asciidoc [hunk 1]
```diff
@@ -18,19 +18,20 @@ refer to
 
 You can create {transform} rules under **{stack-manage-app} > {rules-ui}**.
 
-. On the *Create rule* window, give a name to the rule and optionally provide 
-tags. Select the {transform} health rule type:
+. Click *Create rule* window and select the {transform} health rule type.
```
Comment (lcawl) [inline docs/reference/transform/transform-alerts.asciidoc:21]:
```suggestion
. Click *Create rule* and select the {transform} health rule type.
```

File: docs/reference/transform/transform-alerts.asciidoc [hunk 2]
```diff
@@ -18,19 +18,20 @@ refer to
...
 
-. On the *Create rule* window, give a name to the rule and optionally provide 
-tags. Select the {transform} health rule type:
+. Click *Create rule* and select the {transform} health rule type.
+
+. Give a name to the rule and optionally provide tags.
+
+. Select the {transform} or {transforms} to include. You can also use a special 
+character (`*`) to apply the rule to all your {transforms}. {transforms-cap} 
+created after the rule are automatically included.
```
Comment (przemekwitek) [inline docs/reference/transform/transform-alerts.asciidoc:27]: "rule is" or "rules are"?

Comment (lcawl) [reply docs/reference/transform/transform-alerts.asciidoc:27]: That blurb appears as following in the preview, which I think looks right: "You can also use a special character (*) to apply the rule to all your transforms. Transforms created after the rule are automatically included."

**Review by szabosteve — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**